usmqe/alerting/alerting.py

```diff
@@ -0,0 +1,349 @@
+import pytest
```

Comment:
Do you really need to create new directory/module `alerting` just to have `alerting.py` in it?

Reply:
It can be just a standalone file. I will update pr accordingly.